app/schema/widgets/multiple_choice_widget.py

```diff
@@ -26,3 +26,12 @@ def render(self, answer_state):
         }
 
         return render_template('/partials/widgets/multiple_choice_widget.html', **widget_params)
+
+    def get_other_input(self, post_vars):
+        if len(post_vars) == 0:
+            return None
+
+        other_values = post_vars.getlist(self.name) if hasattr(post_vars, 'getlist') else post_vars.get(self.name)
```

Comment:
Similarly @samiwel, these two if conditions would be better if they were over multiple lines.